src/typescript/02-ide-support/ide-support.md

```diff
@@ -15,7 +15,7 @@ Integrated Development Environments have come a long way in supporting modern we
 
 ### Visual Studio Code
 
-<a href="https://code.visualstudio.com/">VS Code</a> is Microsoft’s modern take on an IDE for app development (P.S. TypeScript is a Microsoft Open Source project).  VS Code has built in TypeScript support for syntax highlighting, IntelliSense code completion, and linting.
+<a href="https://code.visualstudio.com/">VS Code</a> is Microsoft’s modern take on an IDE for app development (P.S. TypeScript is a Microsoft Open Source project). VS Code has built in TypeScript support for syntax highlighting, IntelliSense code completion, and linting.
```

Comment:
The capitalization throughout this whole page is odd, could you go through the whole page and fix it please?